cn.navclub.dbfx.json

```
@@ -0,0 +1,57 @@
+{
+    "app-id": "cn.navclub.dbfx",
+    "runtime": "org.gnome.Sdk",
+    "runtime-version": "42",
+    "sdk": "org.gnome.Sdk",
+    "command": "v-dbfx",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--filesystem=home"
+    ],
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "/share/vala",
+        "*.la",
+        "*.a"
+    ],
+    "modules": [
+        {
+            "name": "flatpak_mysql",
+            "buildsystem": "cmake-ninja",
+            "cleanup": [
+                "lib",
+                "include"
```

```suggestion
                "/lib",
                "/include"
```
But I don't think this does what you think it does. This is not `make clean`. Here you are just deleting what you installed.

Also you install binaries which that you probably generate an invalid flatpak for `aarch64`

Also libmysql is open source so you should build it from source.